tdp_server/api/v1/endpoints/test_component.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import status
+
+from tdp_server.core.config import settings
+
+
+def test_get_component(client):
+    resp = client.get(f"{settings.API_V1_STR}/service/mock/component/node")
+    assert resp.status_code == status.HTTP_200_OK
+    response = resp.json()
+    assert response["id"] == "node"
+
+
+def test_get_component_does_exist(client):
```

Comment:
The test assert that status code is 404 so:
```suggestion
def test_get_component_does_not_exist(client):
```

Reply:
Done